docs/pages/guides/session-keys.mdx

```diff
@@ -1,637 +1,3 @@
-      { name: "signature", type: "bytes", internalType: "bytes" },
-    ],
-    outputs: [
-      {
-        name: "",
-        type: "uint8",
-        internalType: "enum IOffchainAuthorization.Authorization",
-      },
-    ],
-    stateMutability: "view",
-  },
-  {
-    type: "function",
-    name: "owner",
-    inputs: [],
-    outputs: [{ name: "", type: "address", internalType: "address" }],
-    stateMutability: "view",
-  },
-  {
-    type: "function",
-    name: "permissionedCall",
-    inputs: [{ name: "call", type: "bytes", internalType: "bytes" }],
-    outputs: [{ name: "res", type: "bytes", internalType: "bytes" }],
-    stateMutability: "payable",
-  },
-  {
-    type: "function",
-    name: "renounceOwnership",
-    inputs: [],
-    outputs: [],
-    stateMutability: "nonpayable",
-  },
-  {
-    type: "function",
-    name: "supportsPermissionedCallSelector",
-    inputs: [{ name: "", type: "bytes4", internalType: "bytes4" }],
-    outputs: [{ name: "", type: "bool", internalType: "bool" }],
-    stateMutability: "pure",
-  },
-  {
-    type: "function",
-    name: "transferOwnership",
-    inputs: [{ name: "newOwner", type: "address", internalType: "address" }],
-    outputs: [],
-    stateMutability: "nonpayable",
-  },
-  {
-    type: "event",
-    name: "Clicked",
-    inputs: [
-      {
-        name: "account",
-        type: "address",
-        indexed: true,
-        internalType: "address",
-      },
-    ],
-    anonymous: false,
-  },
-  {
-    type: "event",
-    name: "OwnershipTransferred",
-    inputs: [
-      {
-        name: "previousOwner",
-        type: "address",
-        indexed: true,
-        internalType: "address",
-      },
-      {
-        name: "newOwner",
-        type: "address",
-        indexed: true,
-        internalType: "address",
-      },
-    ],
-    anonymous: false,
-  },
-  {
-    type: "error",
-    name: "AddressEmptyCode",
-    inputs: [{ name: "target", type: "address", internalType: "address" }],
-  },
-  { type: "error", name: "FailedCall", inputs: [] },
-  {
-    type: "error",
-    name: "NotPermissionCallable",
-    inputs: [{ name: "selector", type: "bytes4", internalType: "bytes4" }],
-  },
-  {
-    type: "error",
-    name: "OwnableInvalidOwner",
-    inputs: [{ name: "owner", type: "address", internalType: "address" }],
-  },
-  {
-    type: "error",
-    name: "OwnableUnauthorizedAccount",
-    inputs: [{ name: "account", type: "address", internalType: "address" }],
-  },
-];
-```
-
-```text [.env]
-VITE_PAYMASTER_URL=...
-```
-
-:::
-
-You should now be able to click the "Click" button, which will submit a session key transaction. After a few seconds you should see a link to view the transaction you submitted.
-
-### Fetch Permissions
-
-Coming soon
-::::
+We are actively developing new ways for 3rd party signers to use Smart Wallet. Stay tuned for details by joining our `#smart-wallet` channel in [Discord](https://discord.com/invite/cdp/).
```

Comment:
@ilikesymmetry I found this kinda confusing on re-reading. Thoughts on just saying...
```suggestion
This page is under construction. Stay tuned for details by joining our `#smart-wallet` channel in [Discord](https://discord.com/invite/cdp/).
```